.github/workflows/ci.yml

```diff
@@ -140,12 +140,19 @@ jobs:
 
       - run: |
           if [ "${{ matrix.shell }}" == "default" ]; then
-            dotnet r test --verbose
+            dotnet r test --verbose -- --logger "trx;LogFilePrefix=${{ matrix.os }}-integration-tests"
```

Comment:
This needs to be carried over to the `build` job after merging, along with updating to the latest preview build.